config/ac_jwt.go

```diff
@@ -21,11 +19,5 @@ type JWT struct {
 	Name               string   `hcl:"name,label"`
 	PostParam          string   `hcl:"post_param,optional"`
 	QueryParam         string   `hcl:"query_param,optional"`
-	Remain             hcl.Body `hcl:",remain"`
 	SignatureAlgorithm string   `hcl:"signature_algorithm"`
 }
-
-// HCLBody implements the <Body> interface.
-func (j *JWT) HCLBody() hcl.Body {
```

Comment:
This would break acsetter logic: https://github.com/avenga/couper/blob/fix-interfaces/config/configload/load.go#L203

Reply:
We should add a test to ensure all access controls are loaded properly.